### ISSUE TITLE
Manage subjects - Targeting subjects as a whole

### ISSUE THREAD

**Issue by Rajat-Sen** [StatCan/StatcanPublicationUAT#154]
When the user is on the Manage subjects screen, the list items (subjects) are identified differently then the checkbox(button) which is problematic for the accessible users.

Both should be contained under one element and considered as a button so that when the voiceover/talkback is on, it will identify as one item and when activated it should follow/unfollow a subject.

**Comment by ForumG-SC** [comment]
Done
